typescript/phoenix-sdk/src/utils/seatManager.ts

```diff
@@ -207,18 +207,31 @@ export async function confirmOrCreateClaimSeatIxs(
  * @param market The market's address
  * @returns
  */
-export function findTraderToEvict(
+export async function findTraderToEvict(
   client: Client,
   market: PublicKey
-): PublicKey | void {
+): Promise<PublicKey | void> {
   const marketObject = client.markets.get(market.toBase58());
   const traders = marketObject?.data.traders;
 
+  const seatManagerAddress = getSeatManagerAddress(market);
+  const seatManagerStruct: SeatManagerData = deserializeSeatManagerData(
+    (await client.connection.getAccountInfo(seatManagerAddress)).data
+  );
+
   if (
     traders.size >= Number(marketObject.data.header.marketSizeParams.numSeats)
   ) {
     for (const [traderToEvict, traderState] of traders) {
       if (traderState.baseLotsLocked == 0 && traderState.quoteLotsLocked == 0) {
+        // If trader is a DMM, remove the trader as a DMM before evicting.
```

Comment:
? this isn't what we're doing right? if trader is a DMM they are unevictable so we move on

Reply:
I more meant that note to say "a trader must be removed as DMM before you can evict". I can clarify the wording

Reply:
I didn't go with unevictable because I did not wan to imply that any trader can have a permanent spot (they can be removed as DMMs)